src/common/LoggerInterface.ts

```diff
@@ -0,0 +1,25 @@
+import { LogCallback } from 'winston'
+
+/**
+ * Winston logger allows chaining, but not all loggers do.
+ * In order to allow swapping logger implementations this interface hides this feature.
+ * See {@link LeveledLogMethod}
+ */
+interface LeveledLogMethodNoChaining {
+  (message: string, callback: LogCallback): void
+
+  (message: string, meta: any, callback: LogCallback): void
+
+  (message: string, ...meta: any[]): void
+
+  (message: any): void
+
+  (infoObject: object): void
+}
+
+export interface LoggerInterface {
+  error: LeveledLogMethodNoChaining
```

Comment:
please explain to me why its not `(string)=>void`, or at most `(string, ...any[])`
we don't use any other fancy logging features, like callbacks .

Reply:
changed to string=> void